src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -45,6 +45,27 @@ namespace application_manager {
 
 namespace commands {
 
+namespace {
+/**
+ * @brief Functor for build info string
+ */
+struct InfoAdder : std::unary_function<const RPCParams::value_type&, void> {
```

Comment:
@ZhdanovP `InfoAppender` is slightly better, isn't it?

Reply:
@AByzhynar please take a look a84499c9e385732690dfeca562d496cac9b3d483